src/ui/buttons/message.jsx

```diff
@@ -0,0 +1,26 @@
+/* @flow */
+/** @jsx node */
+
+import { node, type ChildType } from "@krakenjs/jsx-pragmatic/src";
+
+import { CLASS } from "../../constants";
+
+const INITIAL_RESERVED_HEIGHT = "36px";
+
+type MessageProps = {|
+  markup: ?string,
+  position: string,
+|};
+
+export function Message({ markup, position }: MessageProps): ChildType {
+  const reservationDiv = `<div class="${CLASS.BUTTON_MESSAGE_RESERVE}" style="height:${INITIAL_RESERVED_HEIGHT}" ></div>`;
+
+  return (
+    <div
+      class={[CLASS.BUTTON_MESSAGE, `${CLASS.BUTTON_MESSAGE}-${position}`].join(
+        " "
+      )}
+      innerHTML={typeof markup === "string" ? markup : reservationDiv}
+    />
+  );
```

Comment:
Github UI probably won't format this well but I think something like this would be clearer in intention:

```suggestion
  const messageClassNames = [CLASS.BUTTON_MESSAGE, `${CLASS.BUTTON_MESSAGE}-${position}`].join(" ")

  if (!typeof markup === "string") {
    return <div className={messageClassNames}>
      <div className={CLASS.BUTTON_MESSAGE_RESERVE} style={`height:${INITIAL_RESERVED_HEIGHT}`} />
     </div>
  }

  return (
    <div
      className={messageClassNames}
      innerHTML={markup}
    />
  );
```

Reply:
or you could do something like this:
```suggestion
  const messageClassNames = [CLASS.BUTTON_MESSAGE, `${CLASS.BUTTON_MESSAGE}-${position}`].join(" ")
  
  if (!typeof markup === "string") {
    return <div className={`${messageClassNames} $CLASS.BUTTON_MESSAGE_RESERVE}`} style={`height:${INITIAL_RESERVED_HEIGHT}`} />
  }
  
  return (
    <div
      className={messageClassNames}
      innerHTML={markup}
    />
  );
```

Reply:
also also `className -> class` in all my examples i forgot i'm not writing React 😂